### PR TITLE
doc: list pipeline in table and add colab links

### PR DESCRIPTION
**Pull request by FeelyChau**
Add colab link to `INSTALL.md` and `README.md`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,8 @@ open https://pipboard.vercel.app/
 
 You will see a web page prompt in your browser, and there is a MNIST showcase on the home page and play around there. 
 
+You also can try [Pipcook][] online with [Google Colab Notebook](https://colab.research.google.com/github/alibaba/pipcook/blob/master/notebooks/pipcook_image_classification.ipynb).
```
Comment (yorkie) [inline README.md:133]: The colab should not be in the Pipboard section.

File: docs/INSTALL.md [hunk 2]
```diff
@@ -4,6 +4,7 @@ There are different ways to install [Pipcook][]:
 
 - [Install via NPM][]. This is the best approach for most users. It will provide a stable version and pre-built packages are available for most platforms.
 - [Build from source][]. This is best for users who want the latest-and-greatest features and aren’t afraid of running a brand-new code. This is also needed for users who wish to contribute to the project.
+- [Google Colab Notebook](https://colab.research.google.com/github/alibaba/pipcook/blob/master/notebooks/pipcook_image_classification.ipynb) **Google Colab** is a free GPU / TPU training platform provided by Google, which can provide free computing resources for developers。
```
Comment (yorkie) [inline docs/INSTALL.md:7]: Ditto, this is not for installing Pipcook on Colab.

**Review by yorkie — CHANGES_REQUESTED** [review]

**Comment by FeelyChau** [comment]
We need to create some notebooks to let Pipcook run on Colab.

**Comment by yorkie** [comment]
> We need to create some notebooks to let Pipcook run on Colab.

We have written some at https://github.com/alibaba/pipcook/tree/master/notebooks.

**Review by yorkie — APPROVED** [review]

**Pull request merged**